src/code/operation.js

```diff
@@ -65,7 +68,10 @@ export default class WasabeeOp {
 
   store() {
     this.stored = Date.now();
-    localStorage[this.ID] = JSON.stringify(this);
+    const local = this.toJSON();
```

Comment:
Am I not understanding this? why go to JSON before adding the server value?

Reply:
I didn't want to send to the server its own address, that's why I add it after toJSON call and before stringify
but sending it is far simple and it not a big overhead so that's fine